ca/certificate-authority.go

```diff
@@ -325,10 +344,20 @@ func (ca *CertificateAuthorityImpl) IssueCertificate(csr x509.CertificateRequest
 	serialBigInt := big.NewInt(0)
 	serialBigInt = serialBigInt.SetBytes(serialBytes)
 
+	var profile string
+	switch key.(type) {
+	case *rsa.PublicKey, rsa.PublicKey:
```

Comment:
You're quite right. The former issue is me being paranoically accommodating. I'll fix both.